apps/storybook-react/stories/Divider.stories.tsx

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
This seems a bit silly. How do we deal with styled components in the stories? Should we use the same eslint config for the stories that we use in Core React?

Reply:
I vote yes 👍